packages/scripts/src/generateIndex.ts

```diff
@@ -33,22 +38,7 @@ export const PACKAGES = [
 let idx = 0;
 
 export function addPackageToModel(model: ApiModel, data: any) {
-	let apiPackage: ApiPackage;
-	if (data.metadata) {
-		const tsdocConfiguration = new TSDocConfiguration();
-		const tsdocConfigFile = TSDocConfigFile.loadFromObject(data.metadata.tsdocConfig);
-		tsdocConfigFile.configureParser(tsdocConfiguration);
-
-		apiPackage = ApiItem.deserialize(data, {
-			apiJsonFilename: '',
-			toolPackage: data.metadata.toolPackage,
-			toolVersion: data.metadata.toolVersion,
-			versionToDeserialize: data.metadata.schemaVersion,
-			tsdocConfiguration,
-		}) as ApiPackage;
-	} else {
-		apiPackage = ApiItem.deserializeDocgen(data, 'discord.js') as ApiPackage;
-	}
+	const apiPackage: ApiPackage = ApiPackage.loadFromJson(data);
```

Comment:
`loadFromJson()` returns an `ApiPackage` & `addMember()` below takes an `ApiPackage`, so this explicit type of `ApiPackage` may be unnecessary.

```suggestion
	const apiPackage = ApiPackage.loadFromJson(data);
```

Reply:
Can technically remove this variable alltogether if you wanted to